docs/release_guidelines.md

```diff
@@ -0,0 +1,14 @@
+# OCS-CI Releases
+
+OCS-CI releases are generated in conjunction with OCS releases, A release is cut
+as soon as OCS is Generally Availabled and available in open market place.
+
+The release helps to checkout specific versions of tests that were run during
+the release and serves as baseline for the specific version of OCS.
+
+# Release Process
+1. Create a tag specific to commit that matches the latest ocs-ci that was used to run
+the regression tests. 
```

Comment:
Can you please provide more specific example how to create a tag here? So anyone will know what exact command to run in this step?

Reply:
Added the steps